lib/node/nodes/merge.js

```diff
@@ -97,7 +110,7 @@ function setAliasTo(columns, alias, as) {
 
         if (as === 'left') {
             // only set alias to cartodb_id column
-            if (column === 'cartodb_id') {
+            if (/^.*cartodb_id$/.test(column)) {
```

Comment:
you meant `.includes()` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes

Reply:
Not really, I wanted all strings that ended in cartodb_id to catch `cartodb_id` `left_cartodb_id` and `right_cartodb_id`, but now thanks to this I see there is a method called [endsWith](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith) so I'll change it.